app/components/utils/toaster/template.hbs

```diff
@@ -1,5 +1,6 @@
 {{#if @toasts.length}}
   <div class="toasts-container">
```

Comment:
Misschien `.vl-alert-container` om link te maken met `.vl-alert`?

Reply:
eerst dacht ik "ja, logisch", maar eigenlijk gaat het niet echt om een container van alerts, maar om een container van toasts. Op heden zijn toasts css gewijs alerts met wat custom klasses erbovenop :thinking: . Ms naar 26195c9 kijken, dat gaat een overzicht geven.
(Heb in deze story enkel het functionele gedeelte + indeling van componenten aangepakt. Buiten toevoeging van loading-alert, geen veranderingen aan CSS tot nu toe. Meer feedback welkom. Implementatie mogelijk niet voor deze story, eerder [KAS-1349](https://kanselarij.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=KAS&modal=detail&selectedIssue=KAS-1349) )

Reply:
CSS rework van de alerts mee te nemen in KAS-1349